includes/rest-api/class-wp-job-manager-models-settings.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Declaration of our Settings Model
+ *
+ * @package WPJM/REST
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class WP_Job_Manager_Models_Settings
+ */
+class WP_Job_Manager_Models_Settings extends WPJM_REST_Model_Declaration_Settings
```

Comment:
Is the way WPJM defines settings universal enough to warrant the abstraction (`WPJM_REST_Model_Declaration_Settings`/`MT_Model_Declaration_Settings`) for defining its schema in Mixtape?